blueprints/hr_assistant/whatsapp_bot_server.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from flask import Flask, request
+from twilio.twiml.messaging_response import MessagingResponse
+
+from mindmeld.components import NaturalLanguageProcessor
+from mindmeld.components.dialogue import Conversation
+from mindmeld import configure_logs
+
+
+class WhatsappBotServer:
+    """
+    A sample server class for Whatsapp integration with any MindMeld application
```

Comment:
Should you add the necessary documentation here:
run ngrok
add twillio sandbox

etc

Reply:
done